pkg/jx/cmd/create_cluster_gke.go

```diff
@@ -251,42 +245,7 @@ func (o *CreateClusterGKEOptions) createClusterGKE() error {
 	o.InstallFlags.Provider = GKE
 
 	// call jx install
-	installOpts := &InstallOptions{
-		CommonOptions:        o.CommonOptions,
-		GitRepositoryOptions: o.CreateClusterOptions.GitRepositoryOptions,
-		Flags:                o.InstallFlags,
-		CreateJenkinsUserOptions: CreateJenkinsUserOptions{
-			Username: "admin",
-			Password: "admin",
-			CreateOptions: CreateOptions{
-				CommonOptions: CommonOptions{
-					Factory:  o.Factory,
-					Out:      o.Out,
-					Err:      o.Err,
-					Headless: true,
-				},
-			},
-		},
-		CreateEnvOptions: CreateEnvOptions{
-			Options: v1.Environment{
-				ObjectMeta: metav1.ObjectMeta{},
-				Spec: v1.EnvironmentSpec{
-					PromotionStrategy: v1.PromotionStrategyTypeAutomatic,
-				},
-			},
-			PromotionStrategy:      string(v1.PromotionStrategyTypeAutomatic),
-			ForkEnvironmentGitRepo: kube.DefaultEnvironmentGitRepoURL,
-			CreateOptions: CreateOptions{
-				CommonOptions: CommonOptions{
-					Factory:   o.Factory,
-					Out:       o.Out,
-					Err:       o.Err,
-					Headless:  true,
-					BatchMode: true,
-				},
-			},
-		},
-	}
+	installOpts := &o.InstallOptions
 	err = installOpts.Run()
```

Comment:
Does removing this mean we still get default environments created on `jx create cluster gke` and the jenkins API token generated?  I had a hard time getting the environments created without having to add multiple common options which also set headless and batch commands.

Reply:
yeah - seems to work! :)